issues/models.py

```diff
@@ -5,12 +5,17 @@
 
 # Create your models here.
 
-
 class User(models.Model):
     name = models.CharField(max_length=300)
 
-
 class Issue(models.Model):
     title = models.CharField(max_length=100)
     details = models.CharField(max_length=1000, null=True, blank=True)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    date = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, realted_name = 'issues', on_delete=models.CASCADE)
+    xcoord = models.FloatField()
```

Comment:
In my update, I used `lat` and `lon` as field names. Probably `latitude` and `longitude` would have been better, but either way I think being explicit about what the coordinate represents, in the name of the variable, is the important thing.

Reply:
I also used `null=True, blank=True` for `lat` and `lon`, which basically just means blank values are allowed in that column in the database. I figured there can be issues without a specific location, so you'd want it blank.